src/pages/workspaces/workspace/Dashboard/CloudInformation.ts

```diff
@@ -108,7 +108,12 @@ const GoogleCloudInformation = (props: GoogleCloudInformationProps): ReactNode =
               [!storageCost, () => 'Loading last updated...'],
               [
                 !!storageCost?.isSuccess,
-                () => `Updated on ${new Date(storageCost?.lastUpdated ?? '').toLocaleDateString()}`,
+                () => {
+                  if (storageCost?.lastUpdated) {
+                    return `Updated on ${new Date(storageCost?.lastUpdated).toLocaleDateString()}`;
```

Comment:
It might be worth doing something similar at line 131 (I don't know why that doesn't check to see if `lastUpdated` exists).